tests/tests_macvlan_initscripts.yml

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
macvlan is not supported by initscripts. The test needs to be marked as NetworkManager only in ensure_provider_tests.py: https://github.com/linux-system-roles/network/blob/main/tests/ensure_provider_tests.py#L63

Reply:
done~